backend-api/template.yaml

```diff
@@ -982,6 +982,91 @@ Resources:
         - !Ref PermissionsBoundary
         - !Ref AWS::NoValue
 
+  AsyncBiometricTokenFunction:
+    Type: AWS::Serverless::Function
+    DependsOn:
+      - AsyncBiometricTokenLogGroup
+    Metadata:
+      BuildMethod: esbuild
+      BuildProperties:
+        Minify: true
+        Target: es2022
+        Sourcemap: false
+        EntryPoints:
+          - src/functions/asyncBiometricToken/asyncBiometricTokenHandler.ts
+    Properties:
+      AutoPublishAlias: live
+      FunctionName: !Sub ${AWS::StackName}-biometric-token
+      Runtime: nodejs20.x
+      Handler: asyncBiometricTokenHandler.lambdaHandler
+      Events:
+        AsyncBiometricToken:
+          Type: Api
+          Properties:
+            Path: /async/biometricToken
+            Method: post
+            RestApiId: !Ref SessionsApi
+      Role: !GetAtt AsyncBiometricTokenLambdaRole.Arn
+      VpcConfig:
+        SubnetIds:
+          - !ImportValue devplatform-vpc-ProtectedSubnetIdA
+          - !ImportValue devplatform-vpc-ProtectedSubnetIdB
+          - !ImportValue devplatform-vpc-ProtectedSubnetIdC
+        SecurityGroupIds:
+          - !ImportValue devplatform-vpc-AWSServicesEndpointSecurityGroupId
+
+  AsyncBiometricTokenFunctionPermission:
+    Type: AWS::Lambda::Permission
+    Properties:
+      Action: lambda:InvokeFunction
+      FunctionName: !GetAtt AsyncBiometricTokenFunction.Arn
+      Principal: events.amazonaws.com
+      SourceAccount: !Sub ${AWS::AccountId}
+      SourceArn: !Sub arn:aws:execute-api:${AWS::Region}:${AWS::AccountId}:${SessionsApi}/*/*/*
+
+  AsyncBiometricTokenLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      RetentionInDays: 30
+      LogGroupName: !Sub /aws/lambda/${AWS::StackName}-biometric-token
+
+  AsyncBiometricTokenLambdaRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Sub ${AWS::StackName}-biometric-token-lambda
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: sts:AssumeRole
+      Policies:
+        - PolicyName: AsyncBiometricTokenFunctionLoggingPolicy
```

Comment:
nit: should this be kebab case?

Reply:
Only because you've made the RoleName kebab case a few lines above :)

Reply:
Agreed in standup that this approach is consistent with how this is in the rest of the template so leaving as is, tech debt ticket to be raised to fix naming conventions in future